Makefile

```diff
@@ -1,7 +1,7 @@
 SHELL := /bin/bash
 
 .DELETE_ON_ERROR:
-.PHONY: all
+.PHONY: all pipupgrade
```

Comment:
Is it usual to write `pipupgrade` instead of `pip-upgrade` and `processdata` rather than `process-data` in Makefiles?

Reply:
I think I picked that up from examples as the usual way for Makefiles, but I am also every time considering changing it to something more readable.
I guess we can do whatever we feel like anyways.